addon/commands/move-article-command.js

```diff
@@ -0,0 +1,60 @@
+export default class InsertArticleCommand {
+  name = 'move-article';
```

Comment:
command has a different name compared to the classname
from the name of the file, its probably the classname that should change